cmd/kubeadm/app/util/marshal.go

```diff
@@ -42,10 +44,9 @@ func MarshalToYaml(obj runtime.Object, gv schema.GroupVersion) ([]byte, error) {
 // TODO: Is specifying the gv really needed here?
 // TODO: Can we support json out of the box easily here?
 func MarshalToYamlForCodecs(obj runtime.Object, gv schema.GroupVersion, codecs serializer.CodecFactory) ([]byte, error) {
-	const mediaType = runtime.ContentTypeYAML
-	info, ok := runtime.SerializerInfoForMediaType(codecs.SupportedMediaTypes(), mediaType)
+	info, ok := runtime.SerializerInfoForMediaType(codecs.SupportedMediaTypes(), yamlMediaType)
 	if !ok {
-		return []byte{}, errors.Errorf("unsupported media type %q", mediaType)
+		return []byte{}, errors.Errorf("unsupported media type %q", yamlMediaType)
```

Comment:
Is there a reason not to use `runtime.ContentTypeYAML` directly?

Reply:
How do I update here? thanks.

Reply:
@xichengliudui 
instead of declaring the constant `const yamlMediaType = runtime.ContentTypeYAML`
just use `runtime.ContentTypeYAML` in the places where it's needed.
thanks.